### PR TITLE
Translate some of missing entities

### PR DESCRIPTION
**Pull request by krzema12**
Just a bunch of simple changes I managed to do during my time slot during these two weeks. Locally I also started translating IrTry, will come next time.

### CODE REVIEW

File: python/README.md [hunk 1]
```diff
@@ -66,10 +66,14 @@ It will generate various reports and summaries:
 
 ### Test stats
 
-Current status: ![coverage](http://www.yarntomato.com/percentbarmaker/button.php?barPosition=29&leftFill=) (**1685**/5787 passed)
+Current status: ![coverage](http://www.yarntomato.com/percentbarmaker/button.php?barPosition=29&leftFill=) (**1709**/5787 passed)
```
Comment (SerVB) [inline python/README.md:69]: **1709**/5787=0.2953...~=30%, so feel free to increase the percentage ;)

```suggestion
Current status: ![coverage](http://www.yarntomato.com/percentbarmaker/button.php?barPosition=30&leftFill=) (**1709**/5787 passed)
```

Just a thought that maybe we need to find an API that calculates the percentage itself...

**Review by SerVB — APPROVED** [review]
Cool! Looks like we have only few missing declarations left

File: python/README.md (hunk 1 above)
Comment (krzema12) [reply python/README.md:69]: Do we need this percentage? It's false anyway - once we started, there were around 15% already :D I think it's enough to have the number of passing VS. Failing/all tests. Wdyt?

Comment (SerVB) [reply python/README.md:69]: Yeah, I don't mind removing this image

Comment (krzema12) [reply python/README.md:69]: https://github.com/krzema12/kotlin-python/pull/32

**Pull request merged**